test/hotspot/jtreg/runtime/SharedArchiveFile/SharedStrings.java

```diff
@@ -26,17 +26,17 @@
  * @summary Check to make sure that shared strings in the bootstrap CDS archive
  *          are actually shared
  * @requires vm.cds.archived.java.heap
+ * @requires vm.flagless
  * @library /test/lib
- * @modules java.base/jdk.internal.misc
- *          java.management
  * @build SharedStringsWb sun.hotspot.WhiteBox
- * @run driver ClassFileInstaller -jar whitebox.jar sun.hotspot.WhiteBox
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller -jar whitebox.jar sun.hotspot.WhiteBox
  * @run driver SharedStrings
  */
 
 import jdk.test.lib.cds.CDSTestUtils;
 import jdk.test.lib.process.ProcessTools;
 import jdk.test.lib.process.OutputAnalyzer;
+import jdk.test.lib.helpers.ClassFileInstaller;
 
```

Comment:
Includes the changes from [8263412](https://bugs.openjdk.org/browse/JDK-8263412) ( commit [e834f99](https://github.com/openjdk/jdk/commit/e834f99d1dee8070e626fd1a4657e0522a414ae7) ) on this file